### PR TITLE
Remove quotation marks from SCO csv during upload

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/4308

There are cases where we have quotation marks within fields of the SCO file that causes [SmarterCSV](https://www.rubydoc.info/github/tilo/smarter_csv) to throw an invalid quotation mark error. We are using the fix as suggested by [SmarterCSV's](https://www.rubydoc.info/github/tilo/smarter_csv) documentation to fix the nested quotation marks issue that comes up when uploading the SCO csv.

## Acceptance criteria
- [x] Able to upload a SCO file.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sonntag-adam — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: app/controllers/uploads_controller.rb [hunk 1]
```diff
@@ -57,7 +57,12 @@ def new_upload(csv_type)
     upload = Upload.new(csv_type: csv_type)
     upload.skip_lines = defaults(csv_type)['skip_lines']
     upload.col_sep = defaults(csv_type)['col_sep']
-
+    unless defaults(csv_type)['force_simple_split'].nil?
+      upload.force_simple_split = defaults(csv_type)['force_simple_split']
+    end
+    unless defaults(csv_type)['strip_chars_from_headers'].nil?
+      upload.strip_chars_from_headers = defaults(csv_type)['strip_chars_from_headers']
+    end
```
Comment (jholton) [inline app/controllers/uploads_controller.rb]: Looks like `force_simple_split` and `strip_chars_from_headers` default to `nil` anyway, so I don't think you need these conditionals.

File: app/controllers/uploads_controller.rb [hunk 2]
```diff
@@ -83,14 +88,23 @@ def upload_params
     @upload_params ||= params.require(:upload).permit(:csv_type, :skip_lines, :col_sep, :upload_file, :comment)
   end
 
+  def set_options
+    csv_type = @upload.csv_type
+    options = { skip_lines: @upload.skip_lines.try(:to_i), col_sep: @upload.col_sep }
+    unless defaults(csv_type)['force_simple_split'].nil? && defaults(csv_type)['strip_chars_from_headers'].nil?
+      options[:force_simple_split] = defaults(csv_type)['force_simple_split']
+      options[:strip_chars_from_headers] = defaults(csv_type)['strip_chars_from_headers']
+    end
```
Comment (jholton) [inline app/controllers/uploads_controller.rb]: Same comment: Looks like `force_simple_split` and `strip_chars_from_headers` default to `nil` anyway, so I don't think you need these conditionals.

File: app/controllers/uploads_controller.rb (hunk 1 above)
Comment (hughes-dustin) [reply app/controllers/uploads_controller.rb]: Ripped out the conditionals. Thanks!

File: app/controllers/uploads_controller.rb [hunk 3]
```diff
@@ -57,7 +57,9 @@ def new_upload(csv_type)
     upload = Upload.new(csv_type: csv_type)
     upload.skip_lines = defaults(csv_type)['skip_lines']
     upload.col_sep = defaults(csv_type)['col_sep']
-
+    upload.force_simple_split = defaults(csv_type)['force_simple_split']
+    upload.strip_chars_from_headers = defaults(csv_type)['strip_chars_from_headers']
+    # end
```
Comment (jholton) [inline app/controllers/uploads_controller.rb]: Looks like you accidentally left in a comment.

Comment (hughes-dustin) [reply app/controllers/uploads_controller.rb]: So I have. It's gone!

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**